ipn/ipnlocal/local.go

```diff
@@ -4744,6 +4751,32 @@ func exitNodeCanProxyDNS(nm *netmap.NetworkMap, peers map[tailcfg.NodeID]tailcfg
 	return "", false
 }
 
+// wireguardExitNodeDNSResolvers returns the DNS resolvers to use for a
+// WireGuard-only exit node, if it has resolver addresses.
+func wireguardExitNodeDNSResolvers(nm *netmap.NetworkMap, peers map[tailcfg.NodeID]tailcfg.NodeView, exitNodeID tailcfg.StableNodeID) ([]*dnstype.Resolver, bool) {
```

Comment:
bump capver? that way we can turn off the new behavior if it break

Reply:
we can turn it off anyway, the field is optional, we can stop sending it.

Reply:
we can then use it to not send it to old clients that don't support it, still worth bumping the cap ver